.github/workflows/build-kata-static-tarball-ppc64le.yaml

```diff
@@ -167,8 +167,8 @@ jobs:
     steps:
       - name: Prepare the self-hosted runner
         run: |
-            ${HOME}/scripts/prepare_runner.sh
-            sudo rm -rf $GITHUB_WORKSPACE/*
+            "${HOME}/scripts/prepare_runner.sh"
+            sudo rm -rf "$GITHUB_WORKSPACE/*"
```

Comment:
Hi @stevenhorsman ,

I think here and in subsequent occurrences you want `"$GITHUB_WORKSPACE"/*`.

```
$ GITHUB_WORKSPACE=/tmp; ls "$GITHUB_WORKSPACE/*"
ls: cannot access '/tmp/*': No such file or directory
$ GITHUB_WORKSPACE=/tmp; ls "$GITHUB_WORKSPACE"/*
/tmp/foo.txt                 /tmp/qtsingleapp-zoom-18f2b-lockfile  /tmp/values_yaml.BkUvIZ  /tmp/values_yaml.ZRfAp3
/tmp/qtsingleapp-zoom-18f2b  /tmp/values_yaml.9n6WOV               /tmp/values_yaml.Qv6f9B
...
...
```

BTW, `sudo rm -rf "$GITHUB_WORKSPACE/*"` is extremely dangerous. I don't know why shellchek doesn't complain. Maybe it does and is fixed on one of the following commits? :)

Reply:
Thanks for spotting this. I made that mistake elsewhere and thought I'd corrected them. I will do now. Thanks for the review. Any no shellcheck doesn't complain